### PR TITLE
GliderDAC Checks

### PR DESCRIPTION
**Pull request by lukecampbell**

### CODE REVIEW

File: compliance_checker/runner.py [hunk 1]
```diff
@@ -39,6 +41,7 @@ def run_checker(cls, ds_loc, checker_names, verbose, criteria):
         cs = ComplianceCheckerCheckSuite()
         ds = cs.load_dataset(ds_loc)
         score_groups = cs.run(ds, *checker_names)
+        print score_groups
```
Comment (daf) [inline compliance_checker/runner.py]: Debug statement or intentional?

File: compliance_checker/tests/test_cf.py [hunk 2]
```diff
@@ -32,6 +32,7 @@
         'cont_ragged'      : resource_filename('compliance_checker', 'tests/data/cont_ragged.nc'),
         'index_ragged'     : resource_filename('compliance_checker', 'tests/data/index_ragged.nc'),
         'bad_missing_data' : resource_filename('compliance_checker', 'tests/data/bad_missing_data.nc'),
+        'glider_std'       : resource_filename('compliance_checker', 'tests/data/gliders/IOOS_Glider_NetCDF_v2.0.nc'),
```
Comment (daf) [inline compliance_checker/tests/test_cf.py]: doesn't look used in the file

**Comment by daf** [comment]
Other than those extremely minor quibbles, looks great!

**Comment by lukecampbell** [comment]
@daf please merge at your leisure.

File: compliance_checker/glider_dac.py [hunk 3]
```diff
@@ -0,0 +1,723 @@
...
+                    'observation_type',
+                    'platform',
+                    'standard_name',
+                    'units',
+                    'valid_min',
+                    'valid_max'
+                ]
+            }
+        }
+
```
Comment (lukecampbell) [inline compliance_checker/glider_dac.py]: @daf I really like this pattern, think I'm gonna use it more often for validation stuff.

Comment (daf) [inline compliance_checker/glider_dac.py]: @lukecampbell seems legit. Prolly should call it `attrs` instead of fields.

**Pull request merged**